utils/saasherder.py

```diff
@@ -0,0 +1,120 @@
+import yaml
+import json
+
+from utils.oc import OC
+from utils.openshift_resource import OpenshiftResource as OR
+
+
+class SaasHerder():
+    """Wrapper around SaaS deployment actions."""
+
+    def __init__(self, saas_files,
+                 github=None,
+                 gitlab=None,
+                 integration=None,
+                 integration_version=None):
```

Comment:
Those parameters are not really optional since the defaults will break the code

Reply:
github and gitlab may be optional as we move forward, since we will divide this integration into internal and external executions.

regarding the rest, i agree, but would like to postpone the fix for a future iteration if you are ok with that.

Reply:
sure thing